scripts/package.json

```diff
@@ -41,7 +41,7 @@
     "keytar": "^7.7.0",
     "lodash": "^4.17.21",
     "minimist": "^1.2.5",
-    "mongodb-connection-string-url": "^2.2.0",
+    "mongodb-connection-string-url": "^2.3.2",
```

Comment:
I can’t comment on the top-level package-lock.json because the diff is too large for github to render, but you probably want to undo those changes (it’s marking all dev dependencies as `extraneous`, which means that something in your local npm config is set up to only install production deps).

Reply:
Fixed the issue with `npm config set -g production false`.

Reply:
Okay -- as I mentioned, in #2651 I'm updating this to a newer version, so it would be great if you could do that here as well (or remove this from the PR since it seems a bit unrelated) 😊

Reply:
I just aligned this module version with the rest of packages. Your pr will overwrite it yeah, but it is no harm to have this bump here, right?

Reply:
yeah, it’s just a merge conflict :)